lib/factorial.rb

```diff
@@ -1,6 +1,18 @@
 # Computes factorial of the input number and returns it
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the number
```

Comment:
What do you mean by `where n is the number`? Why is the space complexity constant?